### PR TITLE
Improve the Button component

### PR DESCRIPTION
**Pull request by thomasJs8**
Hi, there!
These commits are small issues, but I wanna create this PR.

# Summary
- Removed an unused import statement
- Added a case to test the button attribute

# Details
- Removed an unused import statement
The import statement of the `Container` component is commented out.
If you are working on it, this change isn't needed.

- Added a case to test the button attribute
I wanted to test that the `Button` component is rendered correctly.
So I added the `@testing-library/jest-dom` library, then test that it has a 'button' type.
This may be over-testing.
If you don't need this, I'll remove this.

# Anything else
Nothing.

### CODE REVIEW

File: src/components/Button/Button.spec.tsx [hunk 1]
```diff
@@ -1,10 +1,12 @@
 import React from 'react'
 import { render } from '@testing-library/react'
+import '@testing-library/jest-dom/extend-expect'
```
Comment (diego3g) [inline src/components/Button/Button.spec.tsx:3]: This should be in a global setup file

Comment (thomasJs8) [reply src/components/Button/Button.spec.tsx]: I moved it to a global setup file.

**Comment by thomasJs8** [comment]
@diego3g 
Thank you for commenting.
I created the `tests` folder and the `setupTests.ts` file, then I moved the extend to that file.

Please tell me if you have any other ideas of a folder name, a path of a global setup file, or something else.

**Comment by diego3g** [comment]
@thomasJs8 Actually i think it's great for now. Thanks for the PR.

**Pull request merged**